pkg/tiflash/client.go

```diff
@@ -79,6 +94,40 @@ func (c *Client) SendGetRequest(host string, statusPort int, relativeURI string)
 }
 
 func (c *Client) SendPostRequest(host string, statusPort int, relativeURI string, body io.Reader) ([]byte, error) {
+	err := c.checkValidAddress(fmt.Sprintf("%s:%d", host, statusPort))
+	if err != nil {
+		return nil, err
+	}
+
 	uri := fmt.Sprintf("%s://%s:%d%s", c.httpScheme, host, statusPort, relativeURI)
 	return c.httpClient.WithTimeout(c.timeout).SendRequest(c.lifecycleCtx, uri, http.MethodPost, body, ErrFlashClientRequestFailed, distro.Data("tiflash"))
 }
+
+func (c *Client) getMemebers() ([]topology.StoreInfo, error) {
+	cached, _ := c.cache.Get("tiflash_members")
```

Comment:
Seems that each get resets the TTL according to the [doc](https://github.com/ReneKroon/ttlcache/). Maybe this is not a desired behaviour.

Reply:
That's right. I should set `SkipTTLExtensionOnHit` to true.